app/models/world_location_type.rb

```diff
@@ -1,24 +0,0 @@
-class WorldLocationType
```

Comment:
👍 there was so much unnecessary stuff in here! Really nice to get rid of this stuff